services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -639,7 +656,7 @@ async def _clean_expired_uploads(self):
         1. will try to update the entry from S3 backend if exists
         2. will delete the entry if nothing exists in S3 backend.
         """
-        now = datetime.datetime.utcnow()
+        now = datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
careful here: is the datetime in the database naive or aware? cause you might get issues here. that is why I did not change it before.

Reply:
I'll revert this one since I have no idea.